packages/nightingale-heatmap/README.md

```diff
@@ -0,0 +1,37 @@
+# nightingale-heatmap
+
+[![Published on NPM](https://img.shields.io/npm/v/nightingale-contact-map.svg)](https://www.npmjs.com/package/nightingale-contact-map)
+
+Nightingale Heatmap component is used to generate a heatmap visualisation.
+
+## Usage
+
+The below example is using residues contacts data based on the distance.
+
+```html
+<nightingale-heatmap></nightingale-heatmap>
+```
+
+## API Reference
+
+### Properties
+
+#### `width: number`
+
+The width specifies the width of the heat map.
+
+#### `height: number`
+
+The height specifies the height of the heat map.
+
+#### `data: Array`
+
+The data array is of the following structure.
+
+```javascript
+[
+  [1, 1, 0.2],
```

Comment:
Is the data sparse? So what color would appear in position 2,1?

Reply:
In this case, it is symmetric. I have values only for one half. Speaking about that, I can add an attribute asking if it is symmetric and add the symmetric half depending on that.

Reply:
Cool. I was also going for the possibility of default values. For instance  in a 5x5 matrix the data could be only 1 point`[ [2, 4, 0.8] ]` and then everything else is a `0`. NO need to implement this, just thought of it as a possibility once I saw this structure